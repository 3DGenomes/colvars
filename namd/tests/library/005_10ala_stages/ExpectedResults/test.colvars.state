configuration {
  step           20
  dt 5.000000e-01
}

colvar {
  name d
  x  1.65448632224610e+02
}

harmonic {
  configuration {
    name harmonic1
    centers  1.65000000000000e+02
    centers_raw  1.65000000000000e+02
    stage           10
  }
}

