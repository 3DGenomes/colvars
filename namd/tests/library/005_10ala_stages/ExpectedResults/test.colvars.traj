# step         d                     fa_d                  
           0    1.35204451750158e+02 -2.04451750157801e+00 
           1    1.35120663094183e+02 -1.20663094182760e+00 
           2    1.34966848823870e+02  3.03315117612951e+01 
           3    1.37173902792891e+02  8.26097207108859e+00 
           4    1.39973011076713e+02  1.02698892328681e+01 
           5    1.43429777796027e+02 -2.42977779602737e+01 
           6    1.45075523224073e+02 -1.07552322407298e+01 
           7    1.45924514431297e+02 -1.92451443129679e+01 
           8    1.45404425612425e+02  1.59557438757528e+01 
           9    1.45951339220575e+02  1.04866077942467e+01 
          10    1.47168902971856e+02  2.83109702814380e+01 
          11    1.50267246528701e+02 -2.67246528701037e+00 
          12    1.53089645347341e+02 -8.96453473413032e-01 
          13    1.55746539874212e+02 -2.74653987421235e+01 
          14    1.56534173569934e+02 -5.34173569933984e+00 
          15    1.56912464335236e+02 -9.12464335236479e+00 
          16    1.56621804257072e+02  2.37819574292760e+01 
          17    1.57783367382153e+02  1.21663261784730e+01 
          18    1.59644393365397e+02  2.35560663460311e+01 
          19    1.62864652088035e+02 -8.64652088034660e+00 
          20    1.65448632224610e+02 -4.48632224609952e+00 
