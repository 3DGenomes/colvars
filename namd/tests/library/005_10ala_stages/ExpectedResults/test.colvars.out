colvars: ----------------------------------------------------------------------
colvars: # indexFile = "" [default]
colvars: # analysis = off [default]
colvars: # colvarsTrajFrequency = 1
colvars: # colvarsRestartFrequency = 40000 [default]
colvars: # colvarsTrajAppend = off [default]
colvars: The restart output state file will be "test.colvars.state".
colvars: The final output state file will be "test.colvars.state".
colvars: The trajectory file will be "test.colvars.traj".
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = d
colvars:   Initializing a new "dihedral" component.
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # oneSiteSystemForce = off [default]
colvars:       Initializing atom group "group1".
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # disableForces = off [default]
colvars:       # centerReference = off [default]
colvars:       # rotateReference = off [default]
colvars:       # enableFitGradients = on [default]
colvars:       Atom group "group1" defined, 1 atoms initialized: total mass = 14.007.
colvars:       Initializing atom group "group2".
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # disableForces = off [default]
colvars:       # centerReference = off [default]
colvars:       # rotateReference = off [default]
colvars:       # enableFitGradients = on [default]
colvars:       Atom group "group2" defined, 1 atoms initialized: total mass = 12.011.
colvars:       Initializing atom group "group3".
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # disableForces = off [default]
colvars:       # centerReference = off [default]
colvars:       # rotateReference = off [default]
colvars:       # enableFitGradients = on [default]
colvars:       Atom group "group3" defined, 1 atoms initialized: total mass = 12.011.
colvars:       Initializing atom group "group4".
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # disableForces = off [default]
colvars:       # centerReference = off [default]
colvars:       # rotateReference = off [default]
colvars:       # enableFitGradients = on [default]
colvars:       Atom group "group4" defined, 1 atoms initialized: total mass = 14.007.
colvars:   All components initialized.
colvars:   # width = 1 [default]
colvars:   # lowerBoundary = 0 [default]
colvars:   # lowerWallConstant = 0 [default]
colvars:   # upperBoundary = 0 [default]
colvars:   # upperWallConstant = 0 [default]
colvars:   # expandBoundaries = off [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputSystemForce = off [default]
colvars:   # outputAppliedForce = on
colvars: ----------------------------------------------------------------------
colvars: Collective variables initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new "harmonic" instance.
colvars:   # name = "harmonic1" [default]
colvars:   # colvars = { d }
colvars:   # outputEnergy = off [default]
colvars:   # forceConstant = 10
colvars:   # centers = { 135 }
colvars:   # targetCenters = { 165 }
colvars:   # targetForceConstant = 0 [default]
colvars:   # targetNumSteps = 2
colvars:   # targetNumStages = 10
colvars:   # targetForceExponent = 1 [default]
colvars:   # outputCenters = off [default]
colvars:   # outputAccumulatedWork = off [default]
colvars: ----------------------------------------------------------------------
colvars: Collective variables biases initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars: Collective variables module initialized.
colvars: ----------------------------------------------------------------------
colvars:   Moving restraint stage 1 : setting centers to { 138 } at step 2
colvars:   Moving restraint stage 2 : setting centers to { 141 } at step 4
colvars:   Moving restraint stage 3 : setting centers to { 144 } at step 6
colvars:   Moving restraint stage 4 : setting centers to { 147 } at step 8
colvars:   Moving restraint stage 5 : setting centers to { 150 } at step 10
colvars:   Moving restraint stage 6 : setting centers to { 153 } at step 12
colvars:   Moving restraint stage 7 : setting centers to { 156 } at step 14
colvars:   Moving restraint stage 8 : setting centers to { 159 } at step 16
colvars:   Moving restraint stage 9 : setting centers to { 162 } at step 18
colvars:   Moving restraint stage 10 : setting centers to { 165 } at step 20
colvars: Saving collective variables state to "test.colvars.state".
